From b0f847f4a76c8f609679678f71b57973b156221d Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 8 Dec 2016 19:46:41 +0800
Subject: [PATCH 1327/5242] MLK-13579-1 ARM: dts: imx7ulp: adjust suspend
 ocram location

commit  00043633ad2917d6982f0353decd363a9a6b8f07 from
https://source.codeaurora.org/external/imx/linux-imx.git

Adjust suspend ocram location for i.MX7ULP, since previous
location is used by M4.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 173a3dd..def6b5d 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -117,7 +117,7 @@
 
 	sram: sram@20000000 {
 		compatible = "fsl,lpm-sram";
-		reg = <0x20008000 0x4000>;
+		reg = <0x1fffc000 0x4000>;
 	};
 
 	ahbbridge0: ahb-bridge0@40000000 {
-- 
1.7.9.5

